### ISSUE TITLE
Fix the wording of the rule regarding multiple accounts

### ISSUE THREAD

**Issue by IndexLibrorumProhibitorum** [FAForever/website#533]
The rule on multiple accounts on the website should be reworded:
>Do not abuse by using multiple accounts."

This currently implies that it's fine to have multiple accounts as long as you don't cause abuse.
Suggested to reword it to:  "Do not create more than one account", as that is the intended meaning.

**Comment by fcaps** [comment]
what is wrong with multiple accounts? if they don't abuse the system to boost others, or do some other shenanigans, i am fine with it. 
afaik we can detect a abuser, even if he is using multiple accounts

**Comment by Sheikah45** [comment]
Smurfing is against the rules and we do not allow people to have multiple accounts to create an enjoyable experience for all players.

And there is no reason to have multiple accounts beyond wanting to play at a lower ranking.

**Comment by fcaps** [comment]
> And there is no reason to have multiple accounts beyond wanting to play at a lower ranking.

just because you can't think of one.. 

jfyi, you can change the wording, but I still think this is not the right move to make, there are better solutions to counter smurfing that don't include such "rules".

**Comment by IndexLibrorumProhibitorum** [comment]
Sorry, I didn't mean to start a debate on this topic. This was merely a notification from the moderator team (me included) that the text needs to be reworded to align with the way that the game is being moderated. The meaning was always intended to mean that players may not have more than one account, but the current way it is written invites ambiguity. As such, my request this be changed.

**Comment by fcaps** [comment]
@IndexLibrorumProhibitorum sure, and i don't have any veto right here. 
added it to the snowflakes PR, so you get snowflakes and the changed rule.

**Comment by IndexLibrorumProhibitorum** [comment]
Thank you, I appreciate the help!

**Comment by magge-faf** [comment]
> there are better solutions to counter smurfing

@fcaps I would be interested to hear more about it; please create a new issue about that topic to start a debate about your solutions.

**Comment by fcaps** [comment]
@magge-faf https://github.com/FAForever/server/issues/1001
